src/main/app/src/app/zaken/besluit-create/besluit-create.component.ts

```diff
@@ -53,14 +53,18 @@ export class BesluitCreateComponent implements OnInit, OnDestroy {
         this.fields = [[resultaattypeField], [besluittypeField], [ingangsdatumField], [vervaldatumField], [toelichtingField], [documentenField]];
 
         resultaattypeField.formControl.valueChanges.pipe(takeUntil(this.ngDestroy)).subscribe(value => {
-            if (value) {
-                vervaldatumField.required = (value as Resultaattype).vervaldatumBesluitVerplicht;
+            if (!value) {
+                return;
             }
+            vervaldatumField.required = (value as Resultaattype).vervaldatumBesluitVerplicht;
         });
         ingangsdatumField.formControl.valueChanges.pipe(takeUntil(this.ngDestroy)).subscribe(value => {
             (vervaldatumField as DateFormField).minDate = value;
         });
         besluittypeField.formControl.valueChanges.pipe(takeUntil(this.ngDestroy)).subscribe(value => {
+            if (!value) {
```

Comment:
zet gewoon de if(value){ om het blok ipv een return

Reply:
Andere code style toegepast in opvolgende commit